testkit/src/main/scala/org/bitcoins/testkit/eclair/rpc/EclairRpcTestUtil.scala

```diff
@@ -635,6 +634,21 @@ trait EclairRpcTestUtil extends BitcoinSLogger {
     bitcoindRpc
   }
 
+  def clientInSync(client: EclairRpcClient, bitcoind: BitcoindRpcClient)(
```

Comment:
scaladoc please

Reply:
Can this method be private? IT seems to me that we want people using `awaitEclairInSync()`?